server/config/env-config.js

```diff
@@ -2,11 +2,32 @@ import dotenv from 'dotenv';
 
 dotenv.config();
 const { env } = process;
-const { DATABASE_URL, NODE_ENV, PORT, SECRET } = env;
+const {
+  DATABASE_URL,
+  NODE_ENV,
+  PORT,
+  SECRET,
+  SENDGRID_KEY,
+  ADMIN_EMAIL,
+  SITE_NAME,
+  UI_CLIENT_HOST,
+  API_SERVER_HOST,
+  HEROKU_APP_NAME,
+  AUTH_TOKEN_EXPIRY,
+  VERIFICATION_LINK_EXPIRY,
+} = env;
 
 export default {
   DATABASE_URL,
   NODE_ENV,
   PORT,
-  SECRET
+  SECRET,
```

Comment:
This file is getting a bit repetitive, no?

Reply:
This will be addressed as soon as possible.
We'll probably do a conditional export of vars based on current `NODE_ENV`

Reply:
Please do so - I'm pretty sure the team can come up with a more efficient way than writing out the same list of variables twice. Not required for this PR, but I don't expect to still see this by the end of the week.

Also the existence of one environment variable shouldn't influence the existence of another.